docker-entrypoint.sh

```diff
@@ -1,9 +1,11 @@
 #!/bin/bash
 
+set -e
```

Comment:
This causes the script to finish early when `python /checkdb.py` finishes with a non-0 exit code, meaning new installs fail.

I can't figure out whether that's expected, and whether there's missing documentation for bootstrapping, because the PR has no description, documentation, or visible testing. Is there a public ticket? Or is this an error?

Reply:
I will have a look on this one, thank you.

Reply:
You are right in this point. We actually miss a team and a code review. One person cannot be a developer, self code reviewer and a tester ;) 

I introduced the RC versioning, so the testing should be possible easier now, but we miss people :wink:

Reply:
I resolved the issue in:
https://github.com/riotkit-org/docker-taiga/commit/b35d22da378270d207c2e10a9996947eff11a21f

Reply:
I'm releasing a 1.0.1 as a hotfix release. Thank you for the report.